lib/csv_row_model.rb

```diff
@@ -31,6 +32,7 @@ class Boolean; end unless defined? Boolean
 
   require 'csv_row_model/export'
   require 'csv_row_model/export_collection/csv'
+  require 'csv_row_model/export_single_model/csv'
```

Comment:
your thoughts on moving `export_collection` to `export/collection` and `export_single_model` to `export/single_model`?